adders/eslint/config/adder.ts

```diff
@@ -0,0 +1,110 @@
+import { defineAdderConfig } from "@svelte-add/core";
+import { options } from "./options.js";
+
+export const adder = defineAdderConfig({
+    metadata: {
+        id: "eslint",
+        name: "ESLint",
+        description: "A configurable JavaScript linter",
+        environments: { svelte: true, kit: true },
+        website: {
+            logo: "./eslint.svg",
+            keywords: ["eslint", "code", "linter"],
+            documentation: "https://eslint.org/",
+        },
+    },
+    options,
+    integrationType: "inline",
+    packages: [
+        { name: "eslint", version: "^9.7.0", dev: true },
+        { name: "@types/eslint", version: "^9.6.0", dev: true },
```

Comment:
lol. that was fast. I didn't expect you to have incorporated https://github.com/sveltejs/kit/pull/12496 already :smile:

Reply:
ha! that's actually just pure coincidence - I saw [this on twitter](https://x.com/geteslint/status/1815448820647411737) the other day and added it as part of the initial implementation last night 😆